app/services/notices/setup/recipients-add.service.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+/**
+ * Orchestrates the recipients with additional and selected recipients
+ * @module RecipientsAddService
+ */
+
+/**
+ * Orchestrates the recipients with additional and selected recipients
+ *
+ * Formats the recipients to contain only the selectedRecipients
+ *
+ * @param {module:SessionModel} session - The session instance
+ * @param {object[]} recipientsData
+ * @param {boolean} allRecipients - flag to decide if all recipients are required
+ *
+ * @returns {object[]} - recipients
+ */
+function go(session, recipientsData, allRecipients = true) {
+  const recipients = _additionalRecipients(recipientsData, session)
+
+  if (allRecipients || !session.selectedRecipients) {
+    return recipients
+  }
+
+  return _selectedRecipients(session.selectedRecipients, recipients)
+}
+
+function _additionalRecipients(recipientsData, session) {
```

Comment:
Additional recipients do not have the return period in them. 

We use the determinedReturnsPeriod when sending a notification. But the date from the query when the user has selected a return period. 

There might be a bug here where the download shows a different start/ end period than what is sent in the notification. 

We would need to check what is being sent as the start and end date for the notice. They are definitely different.